registration/forms.py

```diff
@@ -39,6 +40,13 @@ class Meta:
         model = User
         fields = (UsernameField(), "email")
 
+    def clean_username(self):
+        username = self.cleaned_data.get('username')
```

Comment:
I would prefer you just call `lower()` here instead of in the conditional and on the return